Rakefile

```diff
@@ -0,0 +1,6 @@
+# Add your own tasks in files placed in lib/tasks ending in .rake,
+# for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
+
+require_relative 'config/application'
+
+Rails.application.load_tasks
```

Comment:
Hmmm... in its current form this Rakefile doesn't do much, as it only loads tasks from bin/rails. Do we need Rake tasks? If not, might be nice to keep the root directory somewhat clean

Reply:
See above

Reply:
I'd remove this for now, I don't think we need special rake tasks right now

Reply:
db:migrate aborts when no rakefile exists. So I had to add it back into the code.